sdk/maps/ci.yml

```diff
@@ -0,0 +1,40 @@
+trigger:
+  branches:
+    include:
+      - main
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/maps/
+    exclude:
+      - sdk/maps/ci.mgmt.yml
+      - sdk/maps/arm-maps
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/maps/
+    exclude:
+      - sdk/maps/ci.mgmt.yml
+      - sdk/maps/arm-maps
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: maps
+    Artifacts:
+      - name: azure-maps-common
+        safeName: azuremapscommon
+      - name: azure-maps-search
+        safeName: azuremapssearch
+      - name: azure-maps-route
+        safeName: azuremapsroute
+      - name: azure-maps-render
```

Comment:
route and render is not included in this PR, should remove them.

Reply:
Done in 6d38e10696b24275391dce2dedb9e3e26a359a68